MonoGame.Framework/Game.cs

```diff
@@ -74,9 +74,12 @@ public bool WaylandVsync
             }
         }
 
+        // If set to true, try to initialize the Wayland SDL video platform first.
+        internal bool PreferWayland { get; }
+
         partial void PlatformConstruct();
 
-        public Game()
+        public Game(bool preferWayland)
```

Comment:
Should probably make this argument optional

```suggestion
        public Game(bool preferWayland = false)
```